docs/DeveloperGuide.md

```diff
@@ -2,28 +2,131 @@
 
 ## Acknowledgements
 
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the
+original source as well}
 
 ## Design & implementation
 
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
 
+### Help menu feature
```

Comment:
there could be an architecture diagram of the high-level design of the app, of how the different features interact, to make the overall app design clearer